cleansio/explicits/explicits.py

```diff
@@ -0,0 +1,34 @@
+""" Loads list of explicits from an encrypted file """
+
+from Crypto.Cipher import AES
+import base64
+import yaml
+import os
+
+class Explicits():
+
+    def __init__(self):
+        pass
+
+    def get_explicits_set(self):
+        """ Decrypt encrypted list of explicits and return a set """
+        decryptor = AES.new('cleansio_sym_key', AES.MODE_CBC, 'cleansioCensorIV')
+
+        current_path = os.path.dirname(__file__)
+        path_to_enc_file = os.path.join(current_path, '../data/explicits-list')
```

Comment:
Can you put this into a helper, something like
```
@classmethod
def __get_explicits_path(cls):
```

Reply:
What's cls?

Reply:
It's like self, but for class methods. The linter requires you to add it when you provide @classmethod.

If you added the helper as a instance method it would complain because it doesn't use self